meta-lmp-base/recipes-support/compose-apps-early-start/compose-apps-early-start/compose-apps-early-start

```diff
@@ -1,5 +1,22 @@
 #!/bin/sh -e
 
+function start_compose_app() {
+	app=$1
+	cd /var/sota/compose-apps/${app}
+
+	# Temporarily disable e to catch possible issues with the initial up
+	set +e
```

Comment:
I'd recommend using the following trick to handle RC under `set -e`

```
echo "Starting app ${app}"
ret=0
docker-compose up -d || ret=$?
```
This method doesn't need to dis/en e

Reply:
Nice, I knew there was a better way but forgot when doing this.